test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/DateTimePrimitiveTypeReference.cs

```diff
@@ -0,0 +1,268 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Generation date: 10/12/2016 11:43:43 AM
+namespace Microsoft.Test.OData.Services.TestServices.DateTimePrimitiveTypeReference

```

Comment:
Please update the lines which are using this namespace. example: line 42, line57 and so on.